lib/widgets/dashboard_widgets/transaction_info_card.dart

```diff
@@ -0,0 +1,89 @@
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_svg/svg.dart';
+import 'package:seeds/constants/app_colors.dart';
+import '../read_times_tamp.dart';
+import '../transaction_avatar.dart';
+import 'package:seeds/utils/string_extension.dart';
+import 'package:seeds/widgets/read_times_tamp.dart';
+import 'package:seeds/widgets/transaction_avatar.dart';
+import 'package:seeds/i18n/wallet.i18n.dart';
+
+class TransactionInfoCard extends StatelessWidget {
+  final String transactionProfileAccount;
+  final String transactionProfileNickname;
+  final String transactionProfileImage;
+  final String transactionTimestamp;
+  final String transactionAmount;
+  final String transactionTypeIcon;
+  final GestureTapCallback callback;
+
+  const TransactionInfoCard({
+    Key key,
+    @required this.transactionAmount,
+    @required this.callback,
+    @required this.transactionProfileAccount,
+    @required this.transactionProfileNickname,
+    @required this.transactionProfileImage,
+    @required this.transactionTimestamp,
+    @required this.transactionTypeIcon,
+  }) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return InkWell(
```

Comment:
this inkwell is missing Ink. Look at how i do it in explore cards

Reply:
ink was added